### ISSUE TITLE
Enable configuration option to not mark emails as SEEN 

### ISSUE THREAD

**Issue by chadell** [nautobot/nautobot-plugin-circuit-maintenance#71]
### Environment

- Nautobot version: 1.1.2
- nautobot-circuit-maintenance version: 0.1.9

### Proposed Functionality

Offer an optional flag in the plugin config in order to change the default behaviour of change the state of an email notification retrieved to SEEN.

### Use Case

In some use case, it would be desirable that the automation processes do not change the SEEN status of an email.

**Comment by glennmatthews** [comment]
As currently implemented, at least in the GMail case, we do *not* mark emails as Read. I am entirely OK with this behavior. However, we've had a feature request to instead assign a specific tag/label to emails **only after they are successfully processed and parsed** - would that fall within scope of this issue, or shall I open a separate issue for that feature?
